### PR TITLE
with updated html and css files

### PR DESCRIPTION
**Pull request by dj12589**
@minmaxdata can you take a look when you have a moment and see if this looks okay to merge? Thank you!

### CODE REVIEW

File: client/dist/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
+    <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
+  </head>
+  <body>
+    <h1>About this item</h1>
+    <div id="service"></div>
+  </body>
+  <script src="bundle.js"></script>
+  <script>
+  $('a').click((e) => {
```
Comment (minmaxdata) [inline client/dist/index.html]: i assume this is just a test to see if jquery is working?

File: server/server.js [hunk 2]
```diff
@@ -4,8 +4,6 @@ const app = express();
 const port = 3001;
 const db = require('../db/index.js');
 
-app.use(express.static('client/dist'));
```
Comment (minmaxdata) [inline server/server.js]: you may want to use the built in dirname from express in the path of your static files location. the path may change once it deployed to amazon.

File: client/dist/index.html (hunk 1 above)
Comment (dj12589) [reply client/dist/index.html]: @minmaxdata - That piece was to change css styling for the menu bar.

File: server/server.js (hunk 2 above)
Comment (dj12589) [reply server/server.js]: great called, fixed that. Thanks!

**Pull request merged**